netjsonconfig/backends/openwrt/converters/interfaces.py

```diff
@@ -34,12 +34,12 @@ class Interfaces(OpenWrtConverter):
     _custom_protocols = ['ppp']
     _interface_dsa_types = ['loopback', 'ethernet', 'bridge']
 
-    def __set_dsa(self, interface):
+    def __set_dsa_interface(self, interface):
         """
-        changes dsa property to manange new syntax introduced in
-        OpenWrt 21.02 by checking supported types and protocols
+        sets dsa interface property to manage new syntax introduced
+        in OpenWrt 21.02 by checking supported types and protocols
         """
-        self.dsa = (
+        self.dsa_interface = (
```

Comment:
I had to create new property instead of using it as a function because while parsing the proto or type can be popped which can give errors.